<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabCamera</class>
 <widget class="QWidget" name="TabCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="wdgCamInfos" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Framerate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_36">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="grpCamCtrls">
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Resolution</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbCamResolution">
           <item>
            <property name="text">
             <string>160x120</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>320x280</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>640x480</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800x600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1024x768</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1280x1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1920x1080</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>FPS</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cmbCamFPS">
           <item>
            <property name="text">
             <string>30</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Focus</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="sldCamFocus">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Exposure</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="sldCamExposure">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="sldCamBrightness">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSlider" name="sldCamContrast">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Brightness</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Contrast</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Saturation</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Hue</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSlider" name="sldCanSaturation">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSlider" name="sldCamHue">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSlider" name="sldCamGamma">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QSlider" name="sldCamGain">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QSlider" name="sldCamSharpness">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Gamma</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Gain</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Sharpness</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnPlay">
           <property name="text">
            <string>Viewfinder</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/icons/resources/media-playback-start.png</normaloff>
             <normalon>:/icons/resources/media-playback-stop.png</normalon>:/icons/resources/media-playback-start.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSnapshot">
           <property name="text">
            <string>Snapshot</string>
           </property>
           <property name="icon">
            <iconset resource="images.qrc">
             <normaloff>:/icons/resources/camera-photo.png</normaloff>:/icons/resources/camera-photo.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSnapshot</sender>
   <signal>clicked()</signal>
   <receiver>TabCamera</receiver>
   <slot>doSnapshot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doSnapshot()</slot>
 </slots>
</ui>
